### DISCUSSION TITLE
Connect to secure ble

### DISCUSSION THREAD

**Discussion by ZeRo1k** [pschatzmann/ESP32-A2DP#474] [Q&A] [answered]
How can I connect to a password protected Bluetooth speaker with the known password 
Thank you
If I connect the ble speaker with a mobile it wants a four digit password it connects after typing the password
But how can I connect it with esp32

**Comment by pschatzmann** [comment, marked as answer]
https://github.com/pschatzmann/ESP32-A2DP#support

Please check the class documentation: The method name is quite speaking!
